dockerfiles/oidc_testprovider

```diff
@@ -1,7 +1,7 @@
-FROM python:3.8
+FROM python:3.9
 EXPOSE 8080
 COPY testprovider /code/
 WORKDIR /code
 
 RUN pip install -r requirements.txt
-CMD ./bin/run.sh
+ENTRYPOINT ["/code/bin/run.sh"]
```

Comment:
Prior to this, it was using the default ENTRYPOINT which is `/bin/sh -c`. This fixes it so it's now running `/code/bin/run.sh` which execs to `python` and that listens to SIGTERM.